pkg/providers/aws/provider_postgres.go

```diff
@@ -52,7 +52,7 @@ const (
 	defaultAwsBackupRetentionPeriod      = 31
 	defaultAwsDBInstanceClass            = "db.t3.small"
 	defaultAwsEngine                     = "postgres"
-	defaultAwsEngineVersion              = "10.16"
+	DefaultAwsEngineVersion              = "10.16"
```

Comment:
Why is this set to public?

Reply:
We want it exposed in RHOAM

Reply:
What is the reasoning for it being exposed in RHOAM? I am just curious as the instance Version will be exposed in the status.

Reply:
Sure we are looking to solve a timing issue around upgrades to RDS and elasticache. And are looking to have as many tools as possible to help us solve the problem so we don't have to do something like this https://github.com/integr8ly/integreatly-operator/pull/2329 again